client/src/components/AuthenticationForm.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Input from './Input';
+import Button from './Button';
+import styled from '@emotion/styled';
+import colors from '../utils/colorsUtils';
+import PropTypes from 'prop-types';
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  > * {
+    flex-basis: 45%;
+  }
+`;
+
+const InputContainer = styled.div`
+  border: solid 2px ${colors.primaryYellow};
+  border-radius: 24px;
+  background-color: ${colors.primaryDark};
+  display: flex;
+  flex-direction: column;
+  padding: 15px;
+  margin-bottom: 6px;
+
+  justify-content: space-evenly;
+`;
+
+const FormContainer = styled.div`
+  width: 300px;
+`;
+
+const SwitchForm = styled.div`
+  color: ${colors.primaryFont};
+  text-align: right;
+  > * {
+    color: ${colors.primaryYellow};
+  }
+`;
+
+const form = {
```

Comment:
this is so smart 😲  very good idea to use this to switch between the different variations 👍 
(is it okay if I steal it? 🙈 👀 )

Reply:
You are very welcome =) ❤️